tests/test_object_detection.py

```diff
@@ -970,3 +972,10 @@ def test_calculate_true_positives_false_positives(return_false_negative):
         np.testing.assert_array_equal(expected_false_positives, false_positives)
     assert counter_dict[4][0] == 5
     assert counter_dict[0][1] == 4
+
+
+def test_per_class_metrics():
```

Comment:
Can you add a simple test for `get_average_per_class_confusion_matrix()` to make sure the right values are calculated? Otherwise test lgtm

Reply:
Added here: https://github.com/Steven-Yiran/cleanlab/blob/8fb478e87af9a5c27d215eb1b025e176c311143f/tests/test_object_detection.py#L990